### PR TITLE
[OP-158] add uk translation for the certificate page

### PR DESCRIPTION
**Pull request by ihor-romaniuk**
**Description:** add UK translation for the certificate page

**Youtrack:** https://youtrack.raccoongang.com/issue/OP-158

**Post merge:**
- [x] Delete working branch (if not needed anymore)

### CODE REVIEW

File: lms/djangoapps/certificates/views/webview.py [hunk 1]
```diff
@@ -252,6 +253,17 @@ def _update_course_context(request, context, course, course_key, platform_name):
                                                               '{partner_short_name}.').format(
             partner_short_name=context['organization_short_name'],
             platform_name=platform_name)
+        
+    # Translators: this text describes the course duration
+    course_duration = CourseOverview.get_from_id(course.id).effort
```
Comment (arsentur) [inline lms/djangoapps/certificates/views/webview.py:258]: Do we need processing here possible exceptions?
What can happen if course overview record was deleted?

File: lms/djangoapps/certificates/views/webview.py [hunk 2]
```diff
@@ -252,6 +253,17 @@ def _update_course_context(request, context, course, course_key, platform_name):
...
+    if course_duration:
+        if course_duration.strip().isnumeric():
+            num_hour = int(course_duration.strip())
+            context['course_duration'] = translation.ungettext(
```
Comment (arsentur) [inline lms/djangoapps/certificates/views/webview.py:262]: It is correct `ungettext`? Why we don't use `_` from the L55?

**Review by arsentur — CHANGES_REQUESTED** [review]

File: lms/djangoapps/certificates/views/webview.py (hunk 1 above)
Comment (mykola-s) [reply lms/djangoapps/certificates/views/webview.py:258]: I think no.
If `course_duration is None`, then the line will not be in the certificate.
It doesn't throw an Exception.

File: lms/djangoapps/certificates/views/webview.py (hunk 2 above)
Comment (mykola-s) [reply lms/djangoapps/certificates/views/webview.py:262]: `ungettext(singular, plural, number)` is used to handle singular and plural forms.
`ugettext(message)` on line L55

File: lms/djangoapps/certificates/views/webview.py (hunk 1 above)
Comment (arsentur) [reply lms/djangoapps/certificates/views/webview.py:258]: I ask you about what will be if `CourseOverview.get_from_id(course.id)` returns `None`?

Comment (mykola-s) [reply lms/djangoapps/certificates/views/webview.py:258]: I can't imagine a situation where `CourseOverview.get_from_id (course.id) == None`
The course object passed to the method is validated on [line 516](https://github.com/raccoongang/edx-platform/pull/2220/files#diff-f83bab3069b5fdf92fc009745a05605aR516) - it means `course.id` will be correct
The `CourseOverview` string associated with the course is created with each course -- therefore `CourseOverview.get_from_id (course.id)` cannot be None.


`\CourseOverview.get_from_id (course.id).effort` can be `None` -- `effort = TextField (null = True)`
but in this case on the [template](https://github.com/raccoongang/edx-platform/pull/2220/files#diff-f83bab3069b5fdf92fc009745a05605aR259) this line will not show.

Comment (arsentur) [reply lms/djangoapps/certificates/views/webview.py:258]: Ok. Thank you for the explanation.

**Review by arsentur — APPROVED** [review]

**Pull request merged**